[2024-10-23T03:00:46.700+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-23T03:00:46.726+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dataframe_move_dag.move_data_task manual__2024-10-23T03:00:43.961838+00:00 [queued]>
[2024-10-23T03:00:46.753+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dataframe_move_dag.move_data_task manual__2024-10-23T03:00:43.961838+00:00 [queued]>
[2024-10-23T03:00:46.754+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-10-23T03:00:46.773+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): move_data_task> on 2024-10-23 03:00:43.961838+00:00
[2024-10-23T03:00:46.780+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=203) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-10-23T03:00:46.783+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dataframe_move_dag', 'move_data_task', 'manual__2024-10-23T03:00:43.961838+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/simpl_etl.py', '--cfg-path', '/tmp/tmpgaymuy1u']
[2024-10-23T03:00:46.784+0000] {standard_task_runner.py:72} INFO - Started process 207 to run task
[2024-10-23T03:00:46.784+0000] {standard_task_runner.py:105} INFO - Job 57: Subtask move_data_task
[2024-10-23T03:00:46.842+0000] {task_command.py:467} INFO - Running <TaskInstance: dataframe_move_dag.move_data_task manual__2024-10-23T03:00:43.961838+00:00 [running]> on host c85a6004b767
[2024-10-23T03:00:46.953+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dataframe_move_dag' AIRFLOW_CTX_TASK_ID='move_data_task' AIRFLOW_CTX_EXECUTION_DATE='2024-10-23T03:00:43.961838+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-23T03:00:43.961838+00:00'
[2024-10-23T03:00:46.955+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-10-23T03:00:46.955+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-10-23T03:00:46.956+0000] {logging_mixin.py:190} INFO - Current task name:move_data_task state:running start_date:2024-10-23 03:00:46.743419+00:00
[2024-10-23T03:00:46.956+0000] {logging_mixin.py:190} INFO - Dag name:dataframe_move_dag and current dag run status:running
[2024-10-23T03:00:46.957+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-23T03:00:46.961+0000] {logging_mixin.py:190} INFO - Source DataFrame:
[2024-10-23T03:00:46.975+0000] {logging_mixin.py:190} INFO -       Name  Age         City
0    Alice   25     New York
1      Bob   30  Los Angeles
2  Charlie   35      Chicago
[2024-10-23T03:00:46.976+0000] {logging_mixin.py:190} INFO - Destination DataFrame:
[2024-10-23T03:00:46.981+0000] {logging_mixin.py:190} INFO -       Name  Age         City
0    Alice   25     New York
1      Bob   30  Los Angeles
2  Charlie   35      Chicago
[2024-10-23T03:00:46.982+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-10-23T03:00:46.993+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-23T03:00:46.994+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dataframe_move_dag, task_id=move_data_task, run_id=manual__2024-10-23T03:00:43.961838+00:00, execution_date=20241023T030043, start_date=20241023T030046, end_date=20241023T030046
[2024-10-23T03:00:47.005+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-10-23T03:00:47.005+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-10-23T03:00:47.006+0000] {logging_mixin.py:190} INFO - Dag name:dataframe_move_dag queued_at:2024-10-23 03:00:44.005911+00:00
[2024-10-23T03:00:47.006+0000] {logging_mixin.py:190} INFO - Task hostname:c85a6004b767 operator:PythonOperator
[2024-10-23T03:00:47.040+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-23T03:00:47.060+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-23T03:00:47.063+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
